osu.Game/Screens/SelectV2/SongSelect.cs

```diff
@@ -255,7 +255,7 @@ protected override void LoadComplete()
             {
                 logo?.ScaleTo(v.NewValue == Visibility.Visible ? 0f : logo_scale, 400, Easing.OutQuint)
                     .FadeTo(v.NewValue == Visibility.Visible ? 0f : 1f, 200, Easing.OutQuint);
-            }, true);
+            });
```

Comment:
This was causing logo animations to be completely interrupted leaving the logo in weird places and delayed in weird ways when entering the song select screen. Resolving the `OsuLogo` like this out-of-band feels really haphazard. I tried removing its DI caching altogether but it's used in other places and the change ballooned out a bit.